core/mix/generate_release.ex

```diff
@@ -85,6 +89,16 @@ defmodule Mix.Tasks.AntikytheraCore.GenerateRelease do
     end
   end
 
+  if Version.match?(System.version(), "~> 1.10") do
```

Comment:
In order not to forget, TODO comment might be helpful.

```suggestion
  # TODO: Remove this conditional branching after requiring Elixir 1.10+
  if Version.match?(System.version(), "~> 1.10") do
```

Reply:
Added at 4547472.